packages/web/src/components.d.ts

```diff
@@ -848,6 +848,24 @@ export namespace Components {
          */
         "variant": 'colour' | 'white';
     }
+    interface GcdsSiteMenu {
```

Comment:
The `GcdsSiteMenu` was removed so it shouldn't appear in this file. Can you run `npm run build` and commit this file again.